app/cache/get.js

```diff
@@ -0,0 +1,13 @@
+const getCache = require('./get-cache')
+const getCacheValue = require('./get-cache-value')
+
+const get = async (request, key) => {
+  try {
+    return await getCacheValue(getCache(request), key)
```

Comment:
Can remove `await` here.  It's an async function so calling functions can await.

Reply:
Can't remove this -- I also thought same as yerself becuz its in an `async` func but a las, I cannot

Reply:
Interesting, wonder why.  Could it be because there's a nested non-async function?  Doesn't feel that would be the reason though.

Reply:
Yeah wouldn't explain not being able to do like-wise for `return await cache.get(key)` though